influxql/parser.go

```diff
@@ -585,6 +585,10 @@ func (p *Parser) parseSelectStatement(tr targetRequirement) (*SelectStatement, e
 		}
 	})
 
+	if d, _ := stmt.GroupByInterval(); stmt.IsRawQuery && d > 0 {
+		return nil, fmt.Errorf("group by needs at least one aggregate function")
```

Comment:
Perhaps capitalize it, to make it clear it's influxql, and I think "requires" reads better. I.e. "GROUP BY requires at least one aggregate function"